### PR TITLE
Add new testmarks

### PR DESCRIPTION
**Pull request by yadro-vavdeev**
Add new testmarks

Signed-off-by: Vladimir Avdeev <v.avdeev@yadro.com>

### CODE REVIEW

File: pytest_tests/testsuites/services/s3_gate/test_s3_gate.py [hunk 1]
```diff
@@ -133,7 +134,7 @@ def test_s3_buckets(self, client_shell):
                 s3_gate_bucket.head_bucket(self.s3_client, bucket_1)
 
     @allure.title("Test S3 Object API")
-    @pytest.mark.sanity
+    @pytest.mark.nightly
```
Comment (dansingjulia) [inline pytest_tests/testsuites/services/s3_gate/test_s3_gate.py:137]: do we need this mark, if all class was marked as nightly ?

File: pytest_tests/testsuites/services/s3_gate/test_s3_multipart.py [hunk 2]
```diff
@@ -14,6 +14,7 @@ def pytest_generate_tests(metafunc):
         metafunc.parametrize("s3_client", ["aws cli", "boto3"], indirect=True)
 
 
+@pytest.mark.nightly
```
Comment (dansingjulia) [inline pytest_tests/testsuites/services/s3_gate/test_s3_multipart.py:17]: pleace add mark not_sbercloud for this suit

File: pytest_tests/testsuites/services/test_http_gate.py [hunk 3]
```diff
@@ -80,7 +81,7 @@ def test_put_grpc_get_http(self, client_shell):
 
     @allure.link("https://github.com/nspcc-dev/neofs-http-gw#uploading", name="uploading")
     @allure.link("https://github.com/nspcc-dev/neofs-http-gw#downloading", name="downloading")
-    @pytest.mark.sanity
+    @pytest.mark.nightly
```
Comment (dansingjulia) [inline pytest_tests/testsuites/services/test_http_gate.py:84]: do we need this mark, if all class was marked as nightly ?

File: pytest_tests/testsuites/payment/test_balance.py [hunk 4]
```diff
@@ -20,6 +20,7 @@
 DEPOSIT_AMOUNT = 30
 
 
+@pytest.mark.nightly
```
Comment (dansingjulia) [inline pytest_tests/testsuites/payment/test_balance.py:23]: add tag not_sbercloud

File: README.md [hunk 5]
```diff
@@ -136,3 +136,10 @@ the feature/topic you are going to implement.
 # License
 
 - [GNU General Public License v3.0](LICENSE)
+
+## Pytest marks
+
+Custom pytest marks used in tests:
+* `nightly` - Tests must be runs in Nightly testruns.
+* `not_sbercloud` - Tests must be skips on Sbercloud platform.
```
Comment (anatoly-bogatyrev) [inline README.md:144]: Could you please change the name of the marks without mentioning a specific providers (change to not_vm or something like this).
Thanks.

**Review by anatoly-bogatyrev — CHANGES_REQUESTED** [review]

File: pytest_tests/pytest.ini [hunk 6]
```diff
@@ -9,6 +9,9 @@ markers =
 #   special markers
     sanity: small tests subset
     staging: test to be excluded from run in verifier/pr-validation/sanity jobs and run test in staging job
+    nightly: test runs in nightly testrun
+    not_sbercloud: test not supported Sbercloud infrastructure
+    not_docker: development environment not supported
```
Comment (aprasolova) [inline pytest_tests/pytest.ini:14]: I suppoose that "running in docker" is not equal to "running in neofs-dev-env" in general. Could you specify the marker description (or marker itself)?

File: pytest_tests/pytest.ini [hunk 7]
```diff
@@ -9,6 +9,9 @@ markers =
 #   special markers
     sanity: small tests subset
     staging: test to be excluded from run in verifier/pr-validation/sanity jobs and run test in staging job
+    nightly: test runs in nightly testrun
+    not_sbercloud: test not supported Sbercloud infrastructure
```
[the remark this answers is not in the document]
Comment (yadro-vavdeev) [reply pytest_tests/pytest.ini]: I rewrite code with considering your comments

File: pytest_tests/pytest.ini [hunk 8]
```diff
@@ -9,20 +9,34 @@ markers =
...
+    s3_gate_tagging: Tagging S3 gate tests
+    s3_gate_versioning: Versioning S3 gate tests
     curl: tests for HTTP gate with curl utility
     long: long tests (with long execution time)
     node_mgmt: neofs control commands
     session_token: tests for operations with session token
-    acl: tests for basic and extended ACL
+    acl: All tests for ACL
+    acl_basic: tests for basic ACL
+    bearer: tests for ACL with bearer
```
[the remark this answers is not in the document]
Comment (yadro-vavdeev) [reply pytest_tests/pytest.ini]: done

File: pytest_tests/testsuites/services/s3_gate/test_s3_gate.py (hunk 1 above)
Comment (yadro-vavdeev) [reply pytest_tests/testsuites/services/s3_gate/test_s3_gate.py]: ok, I kept old mark name: "sanity"

**Pull request merged**